src/commands/DriveToGateNoCompass.cpp

```diff
@@ -37,7 +37,12 @@ command_t DriveToGateNoCompass::getOutput()
 	{
 		calibrationPoints.clear();
 		double err = calculateHeadingErr();
-		return {.thetaVel = err * angleKP, .xVel = 0};
+		double maxthvel = 0.5;
+		double targthvel = err * angleKP;
+		if (abs(targthvel) > maxthvel) {
+			targthvel = (targthvel > 0) ? maxthvel : -maxthvel;
+		}
```

Comment:
Since it looks like this logic is duplicated in a few places, should it be moved to a more centralized place? Maybe we could do the clamping in `setCmdVel()`.